README.md

```diff
@@ -1,14 +1,25 @@
-# latexify_py
-Generates LaTeX math description from Python functions.
+<h1 align = "center">latexify_py</h1>
+A python pacakge that generates LaTeX math description from Python🐍 functions.
 
-## FAQs
+## FAQs:
 1. Which Python version? ... Using 3.6 AST.
 2. GPT-3? ... No, rule-based.
 
-## Example
+## Installation:
+
+```python
+pip install latexify-py
+```
+
+## Example:
 
 [Google Colaboratory](https://colab.research.google.com/drive/1MuiawKpVIZ12MWwyYuzZHmbKThdM5wNJ?usp=sharing)
 
+## Current Contributors:
+
+<p align = "center"><a href="https://github.com/google/latexify_py/graphs/contributors"><img src="https://contributors-img.web.app/image?repo=google/latexify_py">
```

Comment:
- `align = "..."` --> `align="..."`
- Use Markdown syntax instead of HTML tags as well as possible.
- The image looks wide enough to display. Removing the p tag is better for simplicity.